README.md

```diff
@@ -19,10 +19,9 @@ Apollo has a handful of [cache policies](https://www.apollographql.com/docs/reac
 
 **cache-first:** This is the default. `cache-first` means that apollo looks for data in the cache and only when cache doesn't have data apollo loads the data from network. This loosely translates to - apollo loads data from network only once when query hook is mounted for the first time. After that all re-renders and unmount/mount-again of the hooks use cached data. The data is not reset until the page is refreshed. This is problematic sometimes —
 
-- with filters since when filters are changed we want to refetch and not use existing cached data
```

Comment:
I don't believe this is true - I think that apollo caches queries with different filters seperately. In fact this can be frustrating as it means that filtered queries will very rarely hit the cache.